docs/src/sections/sprints/sprint_5.adoc

```diff
@@ -1,5 +1,110 @@
+Same transakcje były tworzone w taki sposób, że odbiegał on mocno od docelowego rozwiązania. Ich podpisywanie również
+odbywało się nie tak, jak powinno. Celem zadania LP-94 była aktualizacja prototypu w taki sposób, aby obsługiwał
+wszystkie funkcjonalności planowane w finalnej wersji aplikacji. Uwzględniało to również generowanie kluczy i
+podpisywanie transakcji w aplikacji mobilnej. Aby to osiągnąć, stworzony został prototyp aplikacji mobilnej posiadający
+te funkcjonalności. Po aktualizacji prototypu serwera i stworzeniu prototypu aplikacji mobilnej udało się przejść
+przez cały proces transakcji, od utworzenia, przez jej podpisanie, aż do walidacji i propagacji do innych węzłów sieci.
+Zadanie to było bardzo trudne. Wymagało zagłębienia się głęboko w protokół Bitcoina, aby uzyskać zamierzony efekt.
+Na zadanie poświęcono prawie tydzień. Proces nauki i zbierania danych był bardzo czasochłonny, a implementacja
+obu prototypów okazała się skomplikowana poprzez niekompatybilność używanych bibliotek z niektórymi nowymi
+funkcjonalnościami Bitcoina. Biblioteka _BitcoinJ_ niestety nie wspiera _PSBT_<<bip174>>, czyli częściowo podpisanych
+transakcji. Po drugiej stronie biblioteka _BitcoinJS_ po stronie aplikacji mobilnej wspierała już tylko transakcje
+_PSBT_, a wsparcie dla starego formatu zostało wycofane. Stworzenie rozwiązania wymagało ręcznego budowania
+odpowiedniej transakcji po stronie serwera, a następnie użycia bardzo starej wersji biblioteki (z 2019 roku) w aplikacji
+mobilnej. Gdyby _BitcoinJ_ był aktywnie rozwijany, a _PSBT_ byłoby zaimplementowane, ukończenie zadania LP-94 zajęłoby
+ułamek poświęconego czasu. Na szczęście udało się pokonać przeciwności i zakończyć pracę nad prototypem. Zdobyta
+wiedza pozwoliła na dodanie przyszłych zadań mających na celu implementację tej logiki po stronie serwera.
+
+Następne dwa zadania dotyczą ekranu wszystkich płatności. Chciano, aby administrator mógł przeglądać wszystkie płatności
+dokonane przez wszystkich użytkowników, bez uwzględnienia danych poufnych, jak kody tokenów. Zrealizowana została
+część serwerowa rozwiązania. Podobnie jak w przypadku osobistej historii płatności, rozwiązanie wspiera w pełni
+paginację, a także jest przygotowane na dodanie w przyszłości sortowania oraz wyszukiwania. Nie udało się zrealizować
+natomiast odpowiedniego ekranu po stronie aplikacji internetowej. Zadanie LP-106 przeniesione zostało do następnej
+iteracji.
+
+W tej iteracji planowano prace nad portfelem oraz zarządzaniem nim. Rozpoczęto od implementacji części serwerowej.
+Najpierw dodany został punkt końcowy pozwalający na stworzenie nowego portfela. W ramach tego zadania konfiguracji
+została poddana biblioteka _BitcoinJ_, gdyż nie była wcześniej używana. Następnie dodano punkt końcowy pozwalający
+na pobranie statusu portfela. Odpowiedź zawiera informacje o balansie w kanałach, węźle Lightning oraz na adresie
```

Comment:
jaka odpowiedz? poprzednie zdanie jest w czasie przeszłym, tutaj przeskakujesz do teraźniejszego. Według mnie lepiej to zdanie trochę zredagować

Reply:
Zrobiłem tak, bo coś zostało dodane, ale teraz jest dalej takie samo. Czyli zostało coś wykonane, a jest tak jak jest.

Reply:
sprecyzowałem odpowiedź btw

Reply:
chodziło mi o to, żeby wskazać w zdaniu że ta odpowiedz pochodzi od tego endpointa np. coś w stylu "Po wysłaniu do niego zapytania ...".

Reply:
dałem "jego zwracana odpowiedź http", "Po wysłaniu do niego zapytania ..." nie pasuje troche do kontekstu zdania